lib/spree/shipment_notice.rb

```diff
@@ -27,14 +27,20 @@ def locate
     # => true
     def update
       shipment.update_attribute(:tracking, tracking)
+      order = shipment.order
+      result = !order.outstanding_balance?
+
+      if Spree::Config.shipstation_capture_at_notification && !result
+        result = order.capture_payments!
+      end
 
       unless shipment.shipped?
         shipment.reload.ship!
         shipment.touch :shipped_at
         shipment.order.update!
       end
 
-      true
+      result
```

Comment:
There's so much going on in this method that this `result` is only partially representative of what has happened. The shipment would still `ship` even if `result` is shipped.